stock_picking_auto_create_lot/README.rst

```diff
@@ -14,13 +14,13 @@ Stock Picking Auto Create Lot
     :target: http://www.gnu.org/licenses/agpl-3.0-standalone.html
     :alt: License: AGPL-3
 .. |badge3| image:: https://img.shields.io/badge/github-OCA%2Fstock--logistics--workflow-lightgray.png?logo=github
-    :target: https://github.com/OCA/stock-logistics-workflow/tree/11.0/stock_picking_auto_create_lot
+    :target: https://github.com/OCA/stock-logistics-workflow/tree/10.0/stock_picking_auto_create_lot
```

Comment:
It seems you have passed incorrect version information to the command line...

Reply:
fixed, sorry fot the typo :(